service/controller/v10/templates/cloudformation/guest/instance.go

```diff
@@ -16,6 +16,15 @@ const Instance = `{{define "instance"}}
       Tags:
       - Key: Name
         Value: {{ .Instance.Cluster.ID }}-master
+  DockerVolume:
```

Comment:
add separate docker disk, instead using ephemeral disk (only available for `m3`)